demo/index.html

```diff
@@ -131,14 +131,14 @@ <h5 class="title">Contributing</h5>
 
 			<footer class="footer">
 				<section class="container">
-					<p>Designed with ♥ by <a target="_blank" href="http://cjpatoilo.com" title="CJ Patoilo">CJ Patoilo</a>. Licensed under the <a target="_blank" href="https://github.com/milligram/milligram#license" title="MIT License">MIT License</a>.</p>
+					<p>Designed with ♥ by <a target="_blank" href="https://cjpatoilo.com" title="CJ Patoilo">CJ Patoilo</a>. Licensed under the <a target="_blank" href="https://github.com/milligram/milligram#license" title="MIT License">MIT License</a>.</p>
```

Comment:
Ah! Indeed. Good catch. Shall I `git push -f` to the branch for this pull request or?